app/data/container-build/cerc-react-peer/Dockerfile

```diff
@@ -9,6 +9,6 @@ COPY . .
 RUN yarn global add serve
 
 RUN echo "Building react-peer" && \
-    yarn install --ignore-scripts && yarn build --ignore @cerc-io/test-app
+    yarn install --ignore-scripts --network-timeout 50000 && yarn build --ignore @cerc-io/test-app
```

Comment:
This seems odd.

Reply:
ran into this error
https://github.com/yarnpkg/yarn/issues/8242#issuecomment-661057834
when running locally; seems to be a common issue on old, slower machines.
Not married to leaving it in, but suspect someone will run into this error again, though #229 and min machine requirements sort of solves the issue.